CONTRIBUTING.md

```diff
@@ -11,4 +13,48 @@ Although contributing to the project is allowed and even encouraged it has to fo
 
 
 
-This is a *guideline* which means that some exceptions may be made. However it is recommend that you respect it for your contribution to be validated. If you have any other question or think a point is unclear or too limiting feel free to contact an @admin
\ No newline at end of file
+This is a *guideline* which means that some exceptions may be made. However it is recommend that you respect it for your contribution to be validated. If you have any other question or think a point is unclear or too limiting feel free to contact an @admin
+
+## Installing an editing
+
+### Requirements
+
+You will need python 3 and the library `discord.py`.
+
+*(We would also recommend to work on Linux :penguin:)*
+
+### The serious stuff
+
+Fork the repository from the [GitHub page](https://github.com/s0lst1ce/ForeBot). This will make a copy of the repo on your account.
+
+Then, on your computer, go to the directory where you want to install the bot and do this command:
+
+`git clone https://github.com/s0lst1ce/ForeBot.git`
+
+Then:
+
+`cd ForeBot` 
+
+And there you go! This is a fast [explanation](https://discordapp.com/developers/docs/intro#bots-and-apps) about how to "create" a bot form the official discord's doc. You have to make an "app" and then you'll have a token in the "Bot" section of your App you will be able to see/copy your token.
+
+You'll have to copy this token to put it in the file `settings.py`. **Warning! Make sure to don't commit your token, remove it before!** If you accidentally commit it you can regenerate it.
```

Comment:
I know this is still true for the `master` branch however the new upcoming version (`todo_fix`) rather uses an environement variables. Maybe replacing this bit with an explanation (or links) on how to set one would be wiser?

Reply:
I don't know about this feature let's talk about that on discord so I'll understand a bit more ;)